src/index.html

```diff
@@ -4,10 +4,302 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <script type="text/javascript" src="./scripts/main.js"></script>
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header" id="home">
+      <div class="header__content">
+        <div class="header__head">
+          <input
+            class="header__nav-toggler"
+            id="nav-toggler"
+            type="checkbox"
+          >
+          <label class="header__nav-button" for="nav-toggler"></label>
+
+          <a href="#home"><div class="header__logo"></div></a>
+          <a class="header__phone-link" href="tel:+12345555555">
+            <div class="header__phone"></div>
+          </a>
+
+          <nav class="header__nav-burger">
+            <div class="header__nav-content">
+              <ul class="header__nav-list">
+                <li class="header__nav-item">
+                  <a href="#home" class="header__nav-link">home</a></label>
```

Comment:
```suggestion
          <nav class="header__nav-burger nav">
            <div class="nav__content">
              <ul class="nav__list">
                <li class="nav__item">
                  <a href="#home" class="nav__link">home</a></label>
```

FYI. You can use in this case mix of classes